org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+                    "url": "https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/cfitsio-4.2.0.tar.gz",
+                    "disable-http-decompression": true,
+                    "sha256": "eba53d1b3f6e345632bb09a7b752ec7ced3d63ec5153a848380f3880c5d61889"
+                }
+            ]
+        },
+        {
+            "name": "wcslib",
+            "config-opts": [
+                "LIBS=-pthread -lcurl -lm",
+                "--without-pgplot",
+                "--without-cfitsio",
+                "--disable-fortran"
+            ],
+            "cleanup": [
+                "/include",
+                "/lib/*.a",
+                "/lib/*.la",
+                "/lib/pkgconfig"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.atnf.csiro.au/people/mcalabre/WCS/wcslib-8.3.tar.bz2",
+                    "sha256": "431ea3417927bbc02b89bfa3415dc0b4668b0f21a3b46fb8a3525e2fcf614508"
+                }
+            ]
+        },
+        {
+            "name": "gsl",
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://mirror.sergal.org/gnu/gsl/gsl-2.7.1.tar.gz",
+                    "sha256": "dcb0fbd43048832b757ff9942691a8dd70026d5da0ff85601e52687f6deeb34b"
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/pkgconfig",
+                "/lib/*.a",
+                "/lib/*.la",
+                "/share/aclocal",
+                "/share/info",
+                "/share/man"
+            ]
+        },
+        {
+            "name": "fftw3",
+            "buildsystem": "autotools",
+            "build-options": {
+                "arch": {
+                    "x86_64": {
+                        "config-opts": [
+                            "--enable-sse2",
+                            "--enable-avx",
+                            "--enable-avx-128-fma"
+                        ]
+                    },
+                    "aarch64": {
+                        "config-opts": [
+                            "--enable-neon"
+                        ]
+                    }
+                }
+            },
+            "config-opts": [
+                "--enable-float",
+                "--enable-threads",
+                "--enable-openmp",
+                "--disable-fortran"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://fftw.org/fftw-3.3.10.tar.gz",
+                    "sha256": "56c932549852cddcfafdab3820b0200c7742675be92179e59e6215b340e26467"
+                }
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
```

Comment:
Same here. Put everything except /bin in toplevel cleanup.

Reply:
57d8a6a